test/GRAPHFILES

```diff
@@ -56,6 +56,7 @@ studies/shootout/regexdna-redux/regexdna-redux.graph
 studies/shootout/reverse-complement/revcomp.graph
 studies/shootout/spectral-norm/spectralnorm.graph
 studies/shootout/thread-ring/thread-ring.graph
+studies/bigint/bigintops.graph
```

Comment:
Any particular reason for `studies/bigint`? I'd lean towards the existing `test/library/standard/BigInteger` (which also has appropriate skipifs already)

Reply:
No, I can move it there.